packages/atomic/src/components/search-box-suggestions/atomic-search-box-query-suggestions/atomic-search-box-query-suggestions.tsx

```diff
@@ -25,6 +25,12 @@ export class AtomicSearchBoxQuerySuggestions {
 
   @State() public error!: Error;
 
+  /**
+   * The icon that will be passed down to `atomic-icon` to be shown next to each suggestion on the list.
+   * Defaults to the atomic icon.
```

Comment:
What is the `atomic icon` ? 🤔 It should be a bit more precise.

Reply:
If we keep the prop, we can copy the doc from atomic-icon

```
/**
 * The SVG icon to display.
 *
 * - Use a value that starts with `http://`, `https://`, `./`, or `../`, to fetch and display an icon from a given location.
 * - Use a value that starts with `assets://`, to display an icon from the Atomic package.
 * - Use a stringified SVG to display it directly.
 */
```

Reply:
ah ok! I thought we wouldnt want to duplicate, and that we could link the docs! but I'll update!!